features/checkout/ability_to_change_email_during_checkout.feature

```diff
@@ -0,0 +1,28 @@
+@checkout
+Feature: Changing email during checkout with registered email
+    In order to change email during checkout
+    As a customer
+    I want to see email input field when im not logged in
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Mantis blade" priced at "$1200"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a customer "John Doe" identified by an email "john@example.com" and a password "secret"
+
+    @ui
+    Scenario: Seeing email input when i checkout as guest
+        Given I have product "Mantis blade" in the cart
+        When I complete addressing step with email "john@example.com" and "United States" based billing address
+        And I go to the addressing step
+        And I complete addressing step with email "new-email@example.com" and "United States" based billing address
+        And I should be checking out as "new-email@example.com"
+
+    @ui
+    Scenario: Hiding email input when checking as logged used
+        Given I am logged in as "john@example.com"
+        And I have product "Mantis blade" in the cart
+        When I complete addressing step with "United States" based billing address
```

Comment:
This step is unnecessary in this scenario. Maybe you could rewrite the scenario to log in during checkout and then check if after going back to addressing page the email change is impossible.